src/components/EnergySeriesChart.tsx

```diff
@@ -42,21 +42,48 @@ export const EnergySeriesChart = ({ data }: OwnProps) => {
       scales: {
         yAxes: [
           {
+            scaleLabel: {
+              display: data.length > 0,
+              labelString: 'Energy (kWh)',
+              fontColor: '#6B7280',
+              fontFamily: ' Arial, Helvetica, sans-serif',
+              fontStyle: 'bold',
+              fontSize: 12,
+            },
             ticks: {
               beginAtZero: true,
               maxTicksLimit: 6, //To avoid having the y axis to crowded with labels.
             },
             afterFit: function (scaleInstance: any) {
-              scaleInstance.width = 60;
+              scaleInstance.width = 80;
```

Comment:
If in production you feel that the scale label is too separated from the graph (because of the size of the y values), please play around with the `scaleInstance.width` value. 
A value of `70` or so might do the trick.